tests/unit/config/test_configfiles.py

```diff
@@ -1000,6 +1000,28 @@ def test_binding_options_hidden(self):
         assert 'bindings.default' not in text
         assert 'bindings.commands' not in text
 
+    def test_unbind(self):
+        opt = configdata.Option(
+            name='opt', typ=configtypes.Int(), default='def',
+            backends=[usertypes.Backend.QtWebEngine], raw_backends=None,
+            description='Hello World')
+        options = [(None, opt, 'val')]
+        bindings = {'normal': {',x': None},
+                    'caret': {',y': 'message-info caret', ',z': None}}
+
+        writer = configfiles.ConfigPyWriter(options, bindings, commented=False)
+        lines = list(writer._gen_lines())
+
+        assert "# Autogenerated config.py" in lines
+        assert "# config.load_autoconfig()" in lines
+        assert "c.opt = 'val'" in lines
+        assert "# Bindings for normal mode" in lines
```

Comment:
Same as above - you're not actually interested in those things for this particular test, so let's remove them and simplify the test.